src/Microsoft.DotNet.Cli.Utils/Constants.cs

```diff
@@ -20,6 +20,7 @@ internal static class Constants
 
         public static readonly string HostExecutableName = "corehost" + ExeSuffix;
         public static readonly string DefaultConfiguration = "Debug";
+        public static readonly string DefaultPublishConfiguration = "Release";
```

Comment:
Having separate default configurations introduces confusing inconsistencies in the toolchain. 

I don't know whether debug or release should be the default, but I do think we should discuss and pick one for the whole toolchain.